developer-docs-site/docs/standards/object.md

```diff
@@ -0,0 +1,239 @@
+
+[object.png](../../static/img/docs/object.png)
+
+### Object resource group
+
+Object is a container for resources that are stored within a single address. These resources usually represent related data often accessed together and should be stored within a single address for data locality and cost savings.
+When created, an object has a resource group, ObjectGroup, by default:
+```rust
+#[resource_group(scope = global)]
+struct ObjectGroup { }
+```
+
+Each object also has the core ObjectCore resource with fundamental properties:
+```rust
+#[resource_group_member(group = aptos_framework::object::ObjectGroup)]
+struct ObjectCore has key {
+    /// Used by guid to guarantee globally unique objects and create event streams
+    guid_creation_num: u64,
+    /// The address (object or account) that owns this object
+    owner: address,
+    /// Object transferring is a common operation, this allows for disabling and enabling
+    /// transfers. Bypassing the use of a the TransferRef.
+    allow_ungated_transfer: bool,
+    /// Emitted events upon transferring of ownership.
+    transfer_events: event::EventHandle<TransferEvent>,
+}
+```
+
+After creating an object, creators can extend with additional resources. For example, an exchange can create an object for each of its liquidity pools and add a resource to track the pool's liquidity.
+```rust
+#[resource_group_member(group = aptos_framework::object::ObjectGroup)]
+struct LiquidityPool has key {
+    token_a: Object<FungibleAsset>,
+    token_b: Object<FungibleAsset>,
+    reserves_a: u128,
+    reserves_b: u128
+}
+```
+
+token_a and token_b are references to other objects. In this example, they're fungible assets (similar to coins). This will be covered in more detail in the fungible asset standard.
+LiquidityPool resource is part of the ObjectGroup resource group. This means that the LiquidityPool resource is stored in the same storage slot as the ObjectCore resource. This is more storage and gas efficient for reading and writing data.
+
+LiquidityPool resource can be added during construction of the object:
+```rust
+pub fun create_liquidity_pool(
+    token_a: Object<FungibleAsset>,
+    token_b: Object<FungibleAsset>,
+    reserves_a: u128,
+    reserves_b: u128
+): Object<LiquidityPool> {
+    let exchange_signer = &get_exchange_signer();
+    let liquidity_pool_constructor_ref = &object::create_object_from_account(exchange_signer);
+    let liquidity_pool_signer = &object::generate_signer(liquidity_pool_constructor_ref);
+    move_to(liquidity_pool_signer, LiquidityPool {
+        token_a: token_a,
+        token_b: token_b,
+        reserves_a: reserves_a,
+        reserves_b: reserves_b
+    });
+}
+```
+
+[liquidity_pool.png](../../static/img/docs/liquidity_pool.png)
```

Comment:
where is the image... note: I realized we need two versions.

Reply:
What do you mean by two versions?

Reply:
dark and normal. There are two themes on aptos.dev which you should try to view locally. I guess your current images are is difficult to distinguish in a dark background. see `static/img/docs/`

Reply:
I removed the diagram for now. Will follow up later with better ones